bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_gnn.py

```diff
@@ -0,0 +1,259 @@
+# Authors:
+# * Sanjiban Sengupta 01/2023
+# * Lorenzo Moneta    01/2023
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from .. import pythonization
+from cppyy import gbl as gbl_namespace
+
+def getActivationFunction(model):
```

Comment:
It is very nice that you are able to write all these functions without ever needing the `import graph_nets`. This is very good!

But, for readers of the code it will be very hard to understand what your functions expect as input arguments and what they do. Please take some time to write docstrings for all the functions and especially document the structure of the input arguments and return values.

For example in this function I would like to know what is the type of `model`, what kind of methods or attributes are you expecting to use from it (e.g. the `_activation` attribute from below).

Reply:
made the change, thanks!